js/slider.js

```diff
@@ -0,0 +1,46 @@
+const sliderElement = document.querySelector('.ad-form__slider');
+const price = document.querySelector('#price');
+const typeHousingField = document.querySelector('#type');
+
+const typeРousing = {
+  bungalow: 0,
+  flat: 1000,
+  hotel: 3000,
+  house: 5000,
+  palace: 10000
+};
+
+noUiSlider.create(sliderElement, {
+  range: {
+    min: 0,
+    max: 100000,
+  },
+  start: 0,
+  step: 1,
+  connect: 'lower',
+  format: {
+    to: function (value) {
```

Comment:
```suggestion
    to: (value) => {
```
у тебя везде стрелочные функции. так что пусть и тут так будет